api/src/main/java/org/openmrs/obs/handler/BinaryDataHandler.java

```diff
@@ -70,6 +72,11 @@ public Obs getObs(Obs obs, String view) {
 			catch (IOException e) {
 				log.error("Trying to read file: " + file.getAbsolutePath(), e);
 			}
+			}
+			else {
+				log.error("The file must exist!!");
```

Comment:
@HerbertYiga I think the error message should be something like The file does not exist since this method is used in a GET API.

Reply:
hi @reagan-meant   thanks for the thoughts here,in relation to GET,Around here we are trying to post the file  than  retrieving and that means we could throw an error message that relates with the posting cc  @ibacher

Reply:
cc @ibacher